locales/hi/php-inline.json

```diff
@@ -1,6 +1,6 @@
 {
     "accepted": "इस क्षेत्र को स्वीकार किया जाना चाहिए ।",
-    "accepted_if": "This field must be accepted when :other is :value.",
+    "accepted_if": ":Attribute को ज़रूर स्वीकार किया जाना चाहिए जब :other :value हो ।",
```

Comment:
In the inline translation file, you do not need to access the attribute.

For example:

> Main: The :attribute must be accepted when :other is :value.
> Inline: This field must be accepted when :other is :value.